src/core/link-to-dfn.js

```diff
@@ -173,9 +176,8 @@ function findLinkTarget(target, anchor, titleToDfns, possibleExternalLinks) {
       anchor.classList.add("internalDFN");
     }
   }
-  // add a bikeshed style indication of the type of link
   if (!anchor.hasAttribute("data-link-type")) {
-    anchor.dataset.linkType = "dfn";
+    anchor.dataset.linkType = "idl" in dfn.dataset ? "idl" : "dfn";
```

Comment:
Why do we have `data-link-type`? The bikeshed documentation only mentions `data-link-type=biblio` and nothing more...

Reply:
The `data-link-type` classifies things into the appropriate types... do a view-source on https://fetch.spec.whatwg.org and you should see the different types. 

For the `biblio` types, we should add support for that too.

Reply:
Filed https://github.com/w3c/respec/issues/2492 so we don't forget.

Reply:
@saschanaz here is the link types: https://tabatkins.github.io/bikeshed/#link-types

Reply:
Thanks! But funny that the list does not have `biblio`.

Reply:
Does this mean that currently core/webidl is assigning wrong values on data-link-type? It assigns IDL type instead of `dfn`/`idl`.

Reply:
potentially, yes. We are using "idl" as a catch all. However, we should fix that as a followup if it's not compatible.